data/brands/landuse/residential.json

```diff
@@ -5,15 +5,48 @@
     "exclude": {"generic": ["^residential$"]}
   },
   "items": [
+    {
+      "displayName": "Ashford Communities",
+      "id": "ashford-c22914",
+      "locationSet": {"include": ["us-tx.geojson"]},
+      "matchNames": [
+        "ashford"
+      ],
+      "tags": {
+        "landuse": "residential",
+        "brand": "Ashford Communities",
+        "brand:wikidata": "Q134701622",
+        "residential": "apartments"
+      }
+    },
     {
       "displayName": "Avalon",
       "id": "avalon-4e5c5e",
       "locationSet": {"include": ["us"]},
+      "matchNames": [
+        "avalon"
```

Comment:
The matcher already knows about this name from `name`, `brand`, and `displayName`, so we can safely omit it.

Reply:
Yes, you can nix that too. 👍

Reply:
@1ec5 I also nixed the "name" field because none of these complexes are named only the brand. It doesn't make sense to have that there as well as a tag that's set. That has to be wrong. This is a deviation from hotels.

Reply:
You’re right that the correct `name=*` of each of these complexes should ideally include the individual location name (which can also go in `branch=*`). That said, having lived at two of these properties over the years, I’d say it’s not that uncommon for residents to use the brand name in isolation, in the same way you’d refer to an individual hotel by its brand.